src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/UserSearchService.kt

```diff
@@ -72,4 +74,7 @@ class UserSearchService(
     }
     return flowOf()
   }
+
+  suspend fun getUserByUsername(username: String): User? =
```

Comment:
Does the response from this now need to be optional? If not this would also remove need for the let block in the controller